README.md

```diff
@@ -1,5 +1,10 @@
 Open-World
-===========
+### Requirments
```

Comment:
1) `Requirements` is spelt incorrect
2) I prefer the original formatting: Open-World as a title; `Installation` section below.

Reply:
Sorry was a typo. Also, will restore formatting if you want.